protocol/src/main/protobuf/UserBitShared.proto

```diff
@@ -379,7 +379,11 @@ enum CoreOperatorType {
   SHP_SUB_SCAN = 65;
   METADATA_HANDLER = 66;
   METADATA_CONTROLLER = 67;
+  // 68 Reserved for Apache Druid
```

Comment:
Please remove these comments. When plugins are merged, new values will be introduced. Protobufs will be regenerated in both cases.

Reply:
Comments removed and protobufs rebuilt.